Desktop/components/JASP/Widgets/JASPDataView.qml

```diff
@@ -16,6 +16,7 @@ FocusScope
 				property alias mouseArea:				wheelCatcher
 				property bool  doubleClickWorkaround:	true
 				property alias isMainDataViewer:		theView.mainData
+	readonly	property alias editCoordinates:			theView.editCoordinates
```

Comment:
Could be removed, I just added it because it might be useful at some point and its not too much extra code anyway.
I first thought the variable-editor used this too...

Reply:
Im not removing it, it could turn out to be useful